testplan/web_ui/testing/src/AssertionPane/AssertionTypes/DictAssertions/DictBaseAssertion.js

```diff
@@ -1,111 +1,61 @@
-import React, {Component, Fragment} from 'react';
+import React, {Fragment} from 'react';
 import PropTypes from 'prop-types';
 import {css, StyleSheet} from 'aphrodite';
-import {AgGridReact} from 'ag-grid-react';
-
-import {
-  calculateDictGridHeight,
-} from './dictAssertionUtils';
+import MaterialTable from 'material-table';
 
 
 /**
  * Base assertion that are used to render dict-like data.
  * It renders the cells with the following content:
  *
  * {Buttons}
- * {Ag-Grid Table}
+ * {Table}
  *
  */
-class DictBaseAssertion extends Component {
-  constructor(props) {
-    super(props);
-
-    this.onGridReady = this.onGridReady.bind(this);
-  }
-
-  /**
-   * Resize the columns of the grid if the component updated.
-   * @public
-   */
-  componentDidUpdate() {
-    this.sizeToFit();
-  }
-
-  /**
-   * Handler for the grid's onGridReady event. The grid's API is loaded and the
-   * columns are resized.
-   * @param params
-   * @public
-   */
-  onGridReady(params) {
-    this.gridApi = params.api;
-    this.gridColumnApi = params.columnApi;
-    this.sizeToFit();
-  }
-
-  /**
-   * Resize every column to take up all the available space but do not exceed
-   * the width of the grid.
-   * @public
-   */
-  sizeToFit() {
-    if (this.gridApi) this.gridApi.sizeColumnsToFit();
-  }
-
-  render() {
-    const normalRows = this.props.rowData.filter(
-      row => !row.descriptor.isEmptyLine).length;
-    const emptyRows = this.props.rowData.filter(
-      row => row.descriptor.isEmptyLine).length;
-    const height = calculateDictGridHeight(normalRows, emptyRows);
-
-    return (
-      <Fragment>
-        {this.props.buttonGroup}
-        <div
-          className={
-            `ag-theme-balham ${css(styles.isResizable)} ${css(styles.gridFont)}`
+export default function DictBaseAssertion(props) {
+  return (
+    <Fragment>
```

Comment:
Instead of 

```js
<Fragment>
    {/* ... */}
</Fragment>
```

you can do

```js
<>
    {/* ... */}
</>
```

https://reactjs.org/docs/fragments.html#short-syntax

Reply:
updated